docs/source/executing-operations/subscription-support.mdx

```diff
@@ -169,6 +169,10 @@ Your router creates a separate WebSocket connection for each client subscription
 * To disambiguate between `graph-ws` and `graph_ws`:
   * `graph-ws` (with a hyphen `-`) is the name of the [library](https://github.com/enisdenjo/graphql-ws) that uses the recommended `graphql_ws` (with un underscore `_`) WebSocket subprotocol. 
 * Each `path` must be set as an _absolute path_. For example, given `http://localhost:8080/foo/bar/graphql/ws`, set the absolute path as `path: "/foo/bar/graphql/ws"`.
+* The `public_url` must include the configured `path` on the Router. For example, given `path` = `/my_callback` then your `public_url` must be something like `http://localhost:8080/my_callback`.
+If you have a proxy in front of the Router that handles the redirection to the right `path` configured in the Router then you can specify another path for the `public_url`, for example `http://localhost:8080/external_path`.
+Under the hood in the Router we will take this URL and add the subscription id in the url to finally have `http://localhost:8080/external_access/{subscription_id}` and pass it directly to your subgraphs.
+If you didn't specify the `path`, by default it's `/callback` so you'll have to specify it in `public_url`.
```

Comment:
About `server URL`  I was using `public_url` because it's the name of the field in configuration. And for the last bullet, it means we don't concatenate the path to the public_url as it was done before, you now have to specify the full url of this callback endpoint. And so if you didn't specify any `path`, by default it's set to `/callback` so you will have to include it by yourself in `public_url`